packages/next/src/compiled/react-server-dom-webpack/cjs/react-server-dom-webpack-server.node.development.js

```diff
@@ -5331,8 +5331,8 @@ function renderToPipeableStream(model, webpackMap, options) {
   };
 }
 
-function decodeReplyFromBusboy(busboyStream, webpackMap) {
-  var response = createResponse(webpackMap, '');
+function decodeReplyFromBusboy(busboyStream, webpackMap, options) {
+  var response = createResponse(webpackMap, '', options ? options.temporaryReferences : undefined);
```

Comment:
Without this `p next dev test/e2e/app-dir/actions/` was crashing after clicking "submit"

Reply:
Upstreamed via https://github.com/facebook/react/pull/29219